src/ui/components/Breethe/template.hbs

```diff
@@ -29,7 +29,10 @@
       @updateFogEffect={{action updateFogEffect}}
     />
   {{/if}}
-  <FogBackground @intensity={{fogIntensity}} />
+
+  {{#unless appState.isSSR}}
+    <FogBackground @intensity={{fogIntensity}} />
+  {{/unless}}
```

Comment:
I just realized this needs to be rendered in the SSR server as well as otherwise the SSR response looks like this:

![localhost_3000_ iphone x](https://user-images.githubusercontent.com/1510/44019690-d6a8b760-9edf-11e8-8645-5371c039df3d.png)

So this should render in SSR but not try to render the particles (which would of course fail anyway).

Reply:
Oh great catch, totally makes sense. I'll fix it now.